packages/modules/imodel-browser/src/containers/iModelGrid/IModelGrid.tsx

```diff
@@ -63,6 +63,8 @@ export interface IModelGridProps {
     iModels: IModelFull[],
     fetchStatus: DataStatus | undefined
   ) => IModelFull[];
+  /**Component to show when there is no iModel */
+  emptyStateComponent?: JSX.Element;
```

Comment:
Please use `React.ReactNode` for the type of this props, to follow the other packages in this repo.

Reply:
Changed to React.ReactNode